recipes/configure.rb

```diff
@@ -67,6 +67,20 @@
   notifies :restart, "service[#{node['nrpe']['service_name']}]"
 end
 
+if (platform?('fedora') && node['platform_version'].to_i >= 20) || (platform_family?('rhel') && node['platform_version'].to_i == 7)
+  template "/usr/lib/systemd/system/nrpe.service" do
```

Comment:
Travis-CI checks fails (in fact rubocop check) on this line for double quote usage. Can you please change them to single-quote..